crates/wast/src/core/resolve/deinline_import_export.rs

```diff
@@ -12,21 +12,24 @@ pub fn run(fields: &mut Vec<ModuleField>) {
                 }
                 match f.kind {
                     FuncKind::Import(import, exact) => {
-                        item = ModuleField::Import(Import {
+                        item = ModuleField::Import(Imports {
                             span: f.span,
-                            module: import.module,
-                            field: import.field,
-                            item: ItemSig {
+                            items: ImportItems::Single(Import {
```

Comment:
For this I might recommend an `Imports::single(import: Import)` helper constructor, but that's just sort of a stylistic preference